packages/components/src/tag/docs/tag/reactRouterLinks.tsx

```diff
@@ -0,0 +1,35 @@
+import { TagGroup, Tag, TagList, HopperProvider } from "@hopper-ui/components";
+import { createMemoryRouter, RouterProvider, useNavigate } from "react-router-dom";
+
+export default function App() {
+    const router = createMemoryRouter([{
+        path: "/123",
```

Comment:
should this be /1 and /2?

Reply:
it matches what was done in Button so I'll keep it like this.